spyderlib/plugins/__init__.py

```diff
@@ -119,11 +212,21 @@ class SpyderDockWidget(QDockWidget):
 
     plugin_closed = Signal()
 
-    def __init__(self, *args, **kwargs):
-        super(SpyderDockWidget, self).__init__(*args, **kwargs)
+    def __init__(self, title, parent):
+        super(SpyderDockWidget, self).__init__(title, parent)

```

Comment:
Sorry, problems with PyQt5 only happen on mixins. So, there shouldn't be any problem with this one :-)